cognite/neat/graph/loaders/_rdf2dms.py

```diff
@@ -108,7 +110,7 @@ def _load(self, stop_on_exception: bool = False) -> Iterable[dm.InstanceApply |
         for view in self.data_model.views:
             view_id = view.as_id()
             tracker.start(repr(view_id))
-            pydantic_cls, edge_by_properties, issues = self._create_validation_classes(view)  # type: ignore[var-annotated]
+            pydantic_cls, edge_by_type, issues = self._create_validation_classes(view)  # type: ignore[var-annotated]
```

Comment:
Edges are identified by type, it was a mistake to use the property id in the schema. That is just a the name used in the schema, which might not be in the triples at all.